### ISSUE TITLE
About hw1 part3

### ISSUE THREAD

**Issue by captainjtx** [mj-hwang/cs287-advanced-robotics#4]
I am learning myself on this course and you posted the only solution I can find online. However, the objective function for HW1-part3  you posted uses 
```
loss = np.linalg.norm(rewards + self.discount * self.value_fun.get_values(states) - self.value_fun.get_values(next_states))
```

To me, the states and next_states are apparently swapped ...

Interestingly, if I did swap them (as I did before I was looking for solution online ...), my return could not go up ...  I will post my solutions later on. Please let me know if anyone sees this and can give me a clue.

**Comment by captainjtx** [comment]
fyi https://github.com/captainjtx/cs287-advanced-robotics if anyone want to revive the code

**Comment by captainjtx** [comment]
I tried to revive your code and swapped the states correctly, the return does not go up like in my case ...

**Comment by captainjtx** [comment]
Ha, I think I figured out, the objective loss is "unfortunately" should be negative of L2 norm. e.g. the adam is doing gradient ascend instead descend. Now the returns goes up now !
```
-------------------------------
| Average Returns | -1.12e+03 |
| Iteration       | 25        |
-------------------------------
-------------------------------
| Average Returns | -1.15e+03 |
| Iteration       | 30        |
-------------------------------
------------------------------
| Average Returns | -661     |
| Iteration       | 35       |
------------------------------
------------------------------
| Average Returns | -823     |
| Iteration       | 40       |
------------------------------
------------------------------
| Average Returns | -554     |
```
